cfme/tests/automate/test_file_import.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [test_requirements.automate, pytest.mark.tier(3)]
+
+
+@pytest.mark.parametrize("import_data", [("bz_1715396.zip", "BZ_1715396", None)])
```

Comment:
This needs an id, `test_domain_import_file[import_data0]` is not a very useful test function.